cachito/workers/nexus_scripts/pip_before_content_staged.groovy

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Note that this script looks much more like the `docker/configure-nexus.groovy` than the js before_content_staged counterpart